### DISCUSSION TITLE
ConanException: Error 2 while executing cmake --build '<HIDDEN>/orbit/build_default_relwithdebinfo' '--' '-j4'

### DISCUSSION THREAD

**Discussion by juanmitaboada** [google/orbit#4437] [Q&A] [answered]
After pulling the project and trying to compile using gcc8 and clang 7 I got at the beginning the error:

**ERROR: Error reading 'default_relwithdebinfo' profile: Profile not found: gcc8_relwithdebinfo**

Which I resolved duplicating clang9* and gcc9* profiles at .conan/profiles/ folder as clang8* and gcc8 profiles.

**Then I tried to compile all again and after several hours and 85 Gbytes of dependencies built I got stuck with the error:**

`[ 26%] Built target ObjectUtils
make: *** [Makefile:95: all] Error 2
ERROR: conanfile.py (OrbitProfiler/None): Error in build() method, line 172
	cmake.build(target=str(self.options.build_target) if self.options.build_target else None)
	ConanException: Error 2 while executing cmake --build '<HIDDEN>/orbit/build_default_relwithdebinfo' '--' '-j4'
`

**Is Orbit really compiling in gcc8 and clang7 as described in DEVELOPMENT.md ?**

> To build Orbit you need a compiler capable of C++17. The following ones should be fine.
>
> - GCC 8 and above on Linux
> - Clang 7 and above on Linux
> - MSVC 2017, 2019 and above on Windows`

What should I do next? (Here is [orbit.log](https://github.com/google/orbit/files/6757933/orbit.log) with the output from compilation)

Thank you,

**Comment by beckerhe** [comment, marked as answer]
Hi Juanmi,
I'm sorry, you're right. We don't actively test for GCC8 compatibility anymore. I will update the DEVELOPMENT.md file.

And it seems you ran into a GCC8-specific problem. All the other compilers should work though. Can you try one of the others, maybe clang9 if possible?

`./build.sh clang9_release` should be one of the fastest builds. It also does not need to much disk space due to not having debug symbols.

Cheers,
Henning
